app/models/transaction.rb

```diff
@@ -0,0 +1,21 @@
+class Transaction < ActiveRecord::Base
```

Comment:
This is probably not a good name. Also, you may have redefined AR's `Transaction` here...